traffic_portal/app/src/modules/private/cdns/config/config.tpl.html

```diff
@@ -26,98 +26,272 @@
+                            <td data-search="^{{::configChange.path}}$">{{::configChange.path}}</td>
+                            <td class="value" data-search="^{{::configChange.oldValue}}$"><json-formatter open="expandLevel" json="configChange.oldValue"></json-formatter></td>
+                            <td class="value" data-search="^{{::configChange.value}}$"><json-formatter open="expandLevel" json="configChange.value"></json-formatter></td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </uib-tab>
+            <uib-tab index="1" class="tab">
+                <uib-tab-heading uib-popover-template="contentRoutersCount.templateUrl" popover-title="{{contentRoutersCount.added + contentRoutersCount.removed + contentRoutersCount.updated}} Total Changes" popover-trigger="mouseenter" popover-placement="top" popover-append-to-body="true">
+                    Traffic Routers<br/>[ {{contentRoutersCount.added}} | {{contentRoutersCount.removed}} | {{contentRoutersCount.updated}} ]
                 </uib-tab-heading>
-                <pre id="config"></pre>
+                <div class="x_content">
+                    <br>
+                    <table id="contentRoutersChangesTable" class="table responsive-utilities jambo_table changes">
+                        <thead>
+                        <tr class="headings">
+                            <th>Change Type</th>
+                            <th>Path</th>
+                            <th>Current Value</th>
+                            <th>Pending Value</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        <tr ng-repeat="cr in ::contentRoutersChanges track by $index">
+                            <td data-search="^{{::cr.op}}$">{{::cr.op}}</td>
+                            <td data-search="^{{::cr.path}}$">{{::cr.path}}</td>
+                            <td class="value" data-search="^{{::cr.oldValue}}$"><json-formatter open="expandLevel" json="cr.oldValue"></json-formatter></td>
+                            <td class="value" data-search="^{{::cr.value}}$"><json-formatter open="expandLevel" json="cr.value"></json-formatter></td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </div>
             </uib-tab>
-            <uib-tab index="1" class="tab" ng-click="diffContentRouters(500)">
-                <uib-tab-heading uib-popover-template="contentRoutersCount.templateUrl" popover-title="{{contentRoutersCount.added + contentRoutersCount.removed}} Total Changes" popover-trigger="mouseenter" popover-placement="top" popover-append-to-body="true">
-                    Traffic Routers [ {{contentRoutersCount.added}} | {{contentRoutersCount.removed}} ]
+            <uib-tab index="1" class="tab">
```

Comment:
According to [UIB docs](https://angular-ui.github.io/bootstrap/#!#tabs):

> _"`index` - Tab index. Must be unique number or string."_

looks like you re-used `1` though

Reply:
whoops